fr/intro/conventions.rst

```diff
@@ -90,34 +90,29 @@ Conventions pour les Models et les Bases de Données
 Les noms de classe de model sont au pluriel et en CamelCase. People, BigPeople,
 et ReallyBigPeople en sont des exemples.
 
-Les noms de tables correspondant aux models CakePHP sont au pluriel et
-utilisent le caractère souligné (underscore). Les tables correspondantes
-aux models mentionnés ci-dessus seront donc respectivement : ``people``,
-``big_people`` et ``really_big_people``.
+Les noms de tables correspondant aux models CakePHP sont au pluriel et utilisent
+le caractère souligné (underscore). Les tables correspondantes aux models
+mentionnés ci-dessus seront donc respectivement : ``people``, ``big_people`` et
+``really_big_people``.
 
-Note des traducteurs francophones : seul le dernier mot est au pluriel et
-tous les pluriels français ne seront pas compris par CakePHP sans lui indiquer
-précisément (par exemple cheval/chevaux). Voir pour cela le chapitre sur les
-inflexions.
-
-Pour vous assurer de la syntaxe des mots pluriels et singuliers, vous pouvez
-utiliser la bibliothèque utilitaire :php:class:`Inflector`. Voir la
-documentation sur :doc:`/core-libraries/inflector` pour plus
-d'informations.
+Note des traducteurs francophones : seul le dernier mot est au pluriel et tous
```

Comment:
This looks good to me (it might need to be read-proof by a native speaker). I'd put it in a ``.. note :`` admonition as well, so it stands out.